source/Outfit.cpp

```diff
@@ -71,7 +71,7 @@ void Outfit::Load(const DataNode &node)
 		else if(child.Token(0) == "weapon")
 			LoadWeapon(child);
 		else if(child.Token(0) == "ammo" && child.Size() >= 2)
-			ammo = GameData::Outfits().Get(child.Token(1));
+			ammoPair = make_pair(GameData::Outfits().Get(child.Token(1)), 0);
```

Comment:
Should this line also compute a count? Or be removed entirely?

Reply:
This line is here because ammo storage outfits need to know which ammo they’ll remove if they’re sold and the ship would become overcapacity. I set the count to 0 because unless ammo is specified as a child of weapon, it doesn’t actually use the ammo.